src/applications/model/gaming-streaming-server.h

```diff
@@ -131,10 +132,15 @@ class GamingStreamingServer : public Application
   Ptr<Socket>  m_socket; //!< Socket
   Address      m_peerAddress; //!< Remote peer address
   uint16_t     m_peerPort; //!< Remote peer port
-  std::vector<EventId> m_sendEvent; //!< Sent event for next packet in each streams
 
-  std::vector<Ptr<RandomVariableStream> > m_packetSizeVariables;  //!< A list of random number generator for packet size
-  std::vector<Ptr<RandomVariableStream> > m_interArrivalTimesVariables; //!< A list of random number generator for packets inter-arrival time
+  struct TrafficStream
+    {
+      EventId m_sendEvent;  //!< Send event for next packet
```

Comment:
The prefix `m_` is used to indicate member variables. You should avoid this prefix for variables of a struct